CHANGELOG.md

```diff
@@ -10,13 +10,13 @@ This file is influenced by http://keepachangelog.com/.
 ### Changed
 - /departments listing style is now the same as /ministers
 - Upgrade UI kit to v1.7.2
+- Removed summaries from /departments
```

Comment:
Just included it to note the specific requirement that this is temporarily not on this page, despite it likely needing to. This will avoid a future scenario of labelling this as a defect when in fact it's a feature/request.